From 71950a9397ae6f75ba8ab30cc3e8aceabfbaaa5b Mon Sep 17 00:00:00 2001
From: krajagopalan01 <kannanrajagopalan@tataelxsi.co.in>
Date: Mon, 29 Jun 2015 19:22:15 +0000
Subject: [PATCH 1/2] RDKDEV-6414 : Add coverity support for components Reason
 for change: Coverity support was missing in yocto builds Test Procedure:
 check if coverity support is added to the build Risks: None

Change-Id: Ib057ff271da5d4e251e95b70ea1d254f86299164
Signed-off-by: krajagopalan01 <kannanrajagopalan@tataelxsi.co.in>
---
 recipes-qt/qt5/qt5.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-qt/qt5/qt5.inc b/recipes-qt/qt5/qt5.inc
index bb0bcdc..513bad3 100644
--- a/recipes-qt/qt5/qt5.inc
+++ b/recipes-qt/qt5/qt5.inc
@@ -1,7 +1,7 @@
 # Copyright (C) 2012, 2014 O.S. Systems Software LTDA.
 # Copyright (C) 2013 Martin Jansa <martin.jansa@gmail.com>
 
-inherit qmake5
+inherit qmake5 coverity
 
 PACKAGECONFIG_OPENSSL ?= "openssl"
 
-- 
2.4.5

